### PR TITLE
Rector is sponsoring Symfony 7.1, thanks to them \o/

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

/cc @TomasVotruba :house: :hugs: 

Symfony 7.1 is looking for more sponsors!

Feel free to reach me using https://symfony.com/support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +89,5 @@ and supported by [Symfony contributors][19].
 [25]: https://symfony.com/doc/current/contributing/code_of_conduct/care_team.html
 [26]: https://symfony.com/book
 [27]: https://symfony.com/backers
-[28]: https://www.shopware.com
-[29]: https://sulu.io
-[30]: https://les-tilleuls.coop/
-[31]: https://symfony.com/sponsor
+[28]: https://symfony.com/sponsor
+[29]: http://getrector.com
```
Comment (94noni) [inline README.md]: https ?

Comment (nicolas-grekas) [reply README.md]: oops, fixed

**Comment by TomasVotruba** [comment]
Thank you for building an awesome framework 🙏

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**